### DISCUSSION TITLE
How to extend script timeout for slots

### DISCUSSION THREAD

**Discussion by CD-Carlos** [SeleniumHQ/docker-selenium#2434] [Q&A] [answered]
I need to run an huge time script  near to 10 minutes,
The container cfg has
                               "slots[0]"."session"."capabilities"."timeouts": {
                                    "implicit": 0,
                                    "pageLoad": 300000,
                                    "script": 30000
                                }
So I have timeout because de script time setup.

I tried to setup in my app with:
 await driver.manage().setTimeouts({ script : 600000 })

But this not work, because the docker script.

What can I do for change that cfg script timeout?

Thanks
CC

**Comment by VietND96** [comment, marked as answer]
I think you can set env var SE_NODE_SESSION_TIMEOUT in Node, unit is in seconds

**Reply by CD-Carlos** [reply]
Thank you VietND96.

It works very well

CC
